app/assets/stylesheets/custom.css.scss

```diff
@@ -895,6 +896,15 @@ $grayMediumLight: #eaeaea;
   /*float: right;*/
 }
 
+.rounded-box {
+  border: 1px solid $sampleAnswerBorder;
```

Comment:
Name of variable `sampleAnswerBorder` should be written in all lowercase letters with hyphens instead of underscores